src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,60 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from Movie").list();
+        }
+    }
+
+    @Override
+    public Boolean update(Movie movie) {
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        Session session = null;
+        Transaction transaction = null;
+
+        try {
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.update(movie);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("adding " + movie + " into database failed", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+
+        return true;
+    }
+
+    @Override
+    public Boolean delete(Long id) {
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        Session session = null;
+        Transaction transaction = null;
+
+        try {
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.delete(get(id)
+                    .orElseThrow(() -> new RuntimeException("now object with id " + id)));
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("removing movie with id: "
+                    + id + " from database failed", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+
+        return true;
```

Comment:
Returning `true` without checking if the delete was successful might not accurately reflect the operation's success. Consider verifying the delete result before returning.